sass/app.scss

```diff
@@ -0,0 +1,309 @@
+  font-size: 10px;
+  padding-left: 1rem;
+  line-height: inherit;
+}
+
+.red-button {
+  max-width: 1040px;
+  position: flex;
+  flex-wrap: wrap;
+  text-align: center;
+  padding-top: 0.5rem;
+}
+.red-button button {
+  flex: 1 0 calc(25% - 10px);
+  text-align: center;
+  margin: 0.2rem;
+  width: 23%;
+}
+
+.fa-facebook-official {
+  color: #445897;
+}
+
+.fa-rss-square {
+  color: #CB8E31;
+}
+
+.fa-twitter-square{
+  color: #698DC6;
+}
+
+.fa-youtube-square {
+color: #B1372C;
+}
+
+.red-button {
+  color: #DCA7AA;
+}
+
+.red-button button {
+  background-color: #B4292E;
+  border: solid #B4292E;
+}
+
+section .header-pic {
+  max-width: 1040px;
+  position: relative;
+}
+.top-img {
+  width: 100%;
+  border: solid white 10px;
+  height: 500px;
+  align-items: center;
+  margin-top: 0.6rem;
+}
+.top-img-logo {
+  position: absolute;
+  top: 15rem;
+  left: 9rem;
+}
+.icon-selector {
+  text-align: center;
+  color: white;
+  margin-top: 10px;
+
+}
+.icon-selector .dot  {
+margin-right: 10px;
+
+
+}
+.arti {
+  display: flex;
+  flex-wrap: wrap;
+  text-align: left;
+  padding-top: 0.5rem;
+  max-width: 1000px;
+  padding: 10px;
+
+
+}
+.content {
+  flex: 1 0 calc(33.3333% - 50px);
+  text-align: left;
+  margin: 4px;
+  border: solid black 1px;
+  background: white;
+  display: flex;
+  flex-direction: column;
+  padding-right: 10px;
+  padding-left: 10px;
+  border-style: none;
+}
+.content p {
+  flex: 1;
+}
+.content a {
+  display: block;
+  text-align: center;
+  flex-direction: column;
+  color: #B4292E;
+  text-transform: uppercase;
+
+
+}
+.the-arrow {
+position: relative;
+left: -100px;
+top: 0px;
+width: 150px;
+background: red;
+border: solid red 1px;
+}
+
+.the-arrow span {
+  font-size: 12px;
+}
+.fa-caret-right{
+  position: absolute;
+  right: -17px;
+  bottom: -10px;
+  color: red;
+  font-size: 44px;
+
+}
+
+
+.img-grid {
+  overflow: auto;
+  position: relative;
+  margin-top: 20px;
+}
+.img-grid h2 {
+position: absolute;
+color: white;
+}
+
+.steve{
+  font-size: 40px;
+  position: absolute;
+  left: 20%;
+  width: 400px;
+  top: 30%
+}
+
+.tommy {
+font-size: 40px;
+width: 250px;
+
+text-align: center;
+position: absolute;
+top: 25%;
+left: 70%;
+}
+
+.lance {
+  font-size: 40px;
+  width: 250px;
+  text-align: center;
+  position: absolute;
+  top: 7%;
+  left: 3%;
+}
+
+.rodney {
+  font-size: 40px;
+  width: 250px;
+  text-align: center;
+  position: absolute;
+  top: 7%;
+  left: 37%;
+}
+
+
+
+
+
+.img-grid__sec{
+  margin: 3px;
+  border: rgb(240, 240, 240) 5px solid;
+  width: calc(33.33% - 8px);
+  float: left;
+}
+
+.img-grid__sec img {
+  width: 100%;
+}
+
+.img-grid__sec--double {
+  width: calc(66.66% - 8px);
+  float: left;
+  transform: translate(0%, -19.5%)
+}
+
+.img-grid__sec--tall{
+  float: right;
+}
+
+.skateboard-container {
+  background: white;
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 0;
+  border-radius: 3px;
+
+}
+.skateboard {
+  margin: 20px;
+  flex: 1 1 calc(10% - 40px);
```

Comment:
Check this on smaller screens, the shrink is making the skate boards really squished.